modules/swagger-codegen/src/main/resources/Java/libraries/feign/pom.mustache

```diff
@@ -274,6 +279,8 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <swagger-core-version>1.5.15</swagger-core-version>
+    <!-- TODO: fix this to a release version after 2.3.0 of swagger-codegen is released-->
+    <swagger-codegen-version>[2.2.9, 3.0.0)</swagger-codegen-version>
```

Comment:
Need a version range here, since `2.3.0` isn't released yet, and there's no obvious way to get the current version of `swagger-codegen` into a mustache variable. If there's any easy way to dynamically replace this with the maven version, we could do that! 

The common interface should change exceedingly rarely, so I doubt this would be a problem. Would be nice if we could force this to be a release version instead possibly being a snapshot version.